server/src/main/resources/evolutions/default/1.sql

```diff
@@ -42,3 +42,21 @@ CREATE TABLE user_tokens (
 
 CREATE INDEX user_tokens_user_id_index ON user_tokens USING BTREE (user_id);
 
+--contains the status for notifications
+CREATE TABLE user_notifications (
+    user_notification_id UUID NOT NULL,
+    user_id UUID NOT NULL,
+    notification_type TEXT NOT NULL,
+    subject TEXT NOT NULL,
+    message TEXT NOT NULL,
+    status TEXT NOT NULL, -- pending/success/failed,
```

Comment:
please create a type for status. for example
```CREATE TYPE NOTIFICATION_STATUS AS ENUM (
    'PENDING',
    'FAILED',
    'SUCCESS'
);```

Reply:
I asked Cristopher to not use postgres enums for now, we can add those once we are confident that no more changes will be required.